.circleci/config.yml

```diff
@@ -122,6 +138,9 @@ workflows:
       - jest-backend:
           requires:
             - setup
+      - jest-new-backend:
+          requires:
+            - setup
```

Comment:
@YazeedLoonat There is a chance that the setup task is what is causing the issue. I believe the way the new backend it setup now it requires a `yarn install` from the backend_new directory as it is not part of the workspaces in the root `package.json`. This means that there is a separate yarn.lock file and everything. So the two options are making it part of the monorepo integration or make a new "setup" task in this config that just builds the new backend